Models/CourseEnrollments.cs

```diff
@@ -13,14 +13,15 @@ public class CourseEnrollments
 
         [ForeignKey("person_system_identifier")]
         [Required]
-        public string student_id { get; set; }
-
-        [Required]
-        public DateTime course_start { get; set; } = DateTime.Now;
+        public string student_id { get; set; }   
 
         [ForeignKey("subject_id")]
         [Required]
```

Comment:
Required is unnecessary you can also make subjects required by adding a initialising it as not null

e.g

`public Subjects Subjects {get; set; } = null!;`